### ISSUE TITLE
"The Basics" remains in quest log after finishing

### ISSUE THREAD

**Issue by Althaia** [EdenServer/community#2042]
### Checklist

- [ ✓ ] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [ ✓ ] I have provided reproducible steps. (No "as the title says" posts please. Provide reproducible steps even if it seems like a waste of time.)
- [ ✓ ] I have provided my client version in the details. (type /ver into your game window)

### Details

My client version is: Version 30181205_0

I did the entire Rycharde-Valgeir quest line and got the final reward, which is a Tea Set.

However, the quest remains in my Current quests instead of Completed quests.

I've noticed a lot on this server when they say in the guide that "additional dialogue" is optionally available, it's not in Eden, so I didn't think much of the fact that Rycharde defaulted to the text of asking me to go see what Valgeir thinks of his dish; this in spite of having done all the required steps and claiming the reward.

But I was just looking over my quest log and this one isn't marked as Completed.

**Comment by Althaia** [comment]
11/09/2019 Version Update included this:

"Rycharde's questline logic was updated to enable The Basics quest."

But I still have The Basics in my Current quest log even after I've completed it.

So I went back to Rycharde to see if anything had changed with him; it had not. He still just has the default line of asking me to go see what Valgeir thinks of his dish.

So I decided to go see what Valgeir would say. He said the same thing he always says, which is that he's busy.

So then I decided to try completing the quest again by buying a Popoto and trading it to Rycharde.

Nothing happened when I tried to trade it to Rycharde, which makes sense because I've already completed this quest.

However, "The Basicis" is still in my Current quest log, not Completed.

So in conclusion, I don't know what y'all did in the Nov. 9 patch, but nothing's changed for me.

**Comment by KnowOne134** [comment]
Well. It's fixed for future people. As far as people that have done it I can write what a gm can do to fix your log

**Comment by Althaia** [comment]
Ah, I see!

A GM kindly took care of it for me, thanks.
